tools/clang/test/HLSLFileCheck/hlsl/functions/attribute/outputtopology_missing.hlsl

```diff
@@ -0,0 +1,17 @@
+// RUN: %dxc -T hs_6_0 -E Hull %s | FileCheck %s
```

Comment:
Do we want a second run command with `-spirv`? Theoretical, but not likely, concern is that someone will move the check to a place on a DXIL only path, and break SPIR-V again.

Reply:
Done (created as a separate test though since SPIR-V codegen is off by default, so it wouldn't necessarily be enabled for these tests on other builds)